bounce.py

```diff
@@ -390,6 +394,19 @@ def _button_release_cb(self, win, event):
                     self._start_step()
         return True
 
+    def _mouse_motion_cb(self, widget, event):
+        if self.ball.grabbed and self._fraction is not None:
+            if event.is_hint:
+                mouse_x = event.window.get_pointer()[0]
+            else:
+                mouse_x = event.x
+
+                self.ball.move_ball_relative((mouse_x
```

Comment:
Your new lines 405 and 406 trigger new flake8 warnings `W503 line break before binary operator`.